index.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  extends: 'airbnb-base'
```

Comment:
Should be left as `airbnb-base` as we don't need React rules in the base config.